components/CreateForm/UploadPreview.tsx

```
@@ -42,6 +50,189 @@ const UploadPreview = () => {
+            console.log('Cropped image saved successfully');
+          }
+        }, 'image/png');
+      };
+
+      img.onerror = () => {
+        setIsUploading(false);
+        toast.error('Failed to load image for cropping');
+      };
+
+      // Load the image
+      if (previewSrc && previewSrc.startsWith('blob:')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)